packages/docs/pages/reference/elements/ellipsis.mdx

```diff
@@ -28,5 +28,8 @@ This element can appear as an immediate child of the following elements.
 <ParentsDisplay name="ellipsis" />
 
 ## Examples
-
-    
\ No newline at end of file
+```ptx-example
+<FRAGMENT parents="p">
+    I wonder what an ellipsis does <ellipsis/>
```

Comment:
I think the sentence should end in a period and there be no space before the ellipsis. If formatting is contentous, we could use an example like `You'll<ellipsis />never defeate<ellipsis />me! (last words spoken to Batman)`